### ISSUE TITLE
Callback called with set_wakeup_callback() leads to crash (also: callback args)

### ISSUE THREAD

**Issue by ghost** [andre-d/pympv#4]
We talked about this on IRC, so this is just a reminder. It appears that Python crashes mysteriously as soon as the user callback is invoked. I have no idea how to debug this.

Note that the callback is completely asynchronous and from another thread, so maybe it's a synchronization problem.

Also, by the way: why does the function require a callback argument? This makes sense in languages like C which have no closures, but in Python you could just use a lambda. So maybe the argument should just be removed.

**Comment by andre-d** [comment]
Fixed in 3b5cf2f352632b678288c2234baa723cfee5a228, removing the callback data then closing this issue

**Comment by andre-d** [comment]
Removed callback data in 5330497b58072449b1f6a08b9b932f241d21965e

**Comment by ghost** [comment]
Seems to work now. On termination I sometimes get this, though:

```
Exception ignored in: 'mpv._c_callback'
KeyError: (140737205153424,)
```

**Comment by andre-d** [comment]
Fixed in 05689fd296e83ea2b2a080febbf67274ef31bc3a
